### ISSUE TITLE
remove descendants on auth nxdomain

### ISSUE THREAD

**Issue by vttale** [vttale/serve-stale#2]
~~~~
From: Paul Vixie <...@...>
Subject: Re: [DNSOP] draft-tale-dnsop-serve-stale
To: Warren Kumari <...@...>
Cc: dnsop <dnsop@ietf.org>, Pieter Lexis <...@...>
Date: Tue, 28 Mar 2017 10:20:40 -0700

[...]

speaking of resimprove, i hope you'll include in this draft the idea of
using delegation-TTL as a delegation-recheck interval, and using an
authoritative NXDOMAIN from the delegator as proof that you need to run
an "rm -rf" in your cache.

i bring this up because we need to be able to kill more cached data
faster-- the opposite of stretchiness-- for abuse control reasons. if
you're going to soften the signaling for cache expiration, you really
ought to balance that out with this simple method of also hardening it.
~~~~

**Comment by vttale** [comment]
~~~~
From: Dave Lawrence <...@...>
Subject: Re: [DNSOP] draft-tale-dnsop-serve-stale
To: Paul Vixie <...@...>
Cc: dnsop <dnsop@ietf.org>, Pieter Lexis <...@...>
Date: Tue, 28 Mar 2017 17:44:16 -0400

Paul Vixie writes:
> speaking of resimprove, i hope you'll include in this draft the idea of
> using delegation-TTL as a delegation-recheck interval, and using an
> authoritative NXDOMAIN from the delegator as proof that you need to run
> an "rm -rf" in your cache.

I definitely like the latter idea as wholly on-point for the main
issue this draft is trying to address, and will get it into the next
rev.

I'm slightly less sure about dragging delegating rechecking into it,
but open enough to being convinced that it is also on-point and
doesn't unduly complicate what is conceptually very straightforward in
the current draft.

I'm also in favour of revivificating resimprove.  I don't know why it
languished either, but don't recall substantial objection.  If it were
brought back that somewhat obviates needing to bring up the delegation
recheck here.
~~~~

**Comment by vttale** [comment]
Addressed in 0d14facc2d7e4f271dfbda808e0ac3c86867a9cd but not yet closed.  Likely result is to remove the comment that was added.

**Comment by vttale** [comment]
Closed per 0d14facc2d7e4f271dfbda808e0ac3c86867a9cd
